moveit_core/CMakeLists.txt

```diff
@@ -25,12 +22,12 @@ endif()
 
 find_package(Boost REQUIRED system filesystem date_time thread iostreams)
 find_package(Eigen3 REQUIRED)
-find_package(PkgConfig REQUIRED)
 
-pkg_check_modules(LIBFCL REQUIRED fcl)
-# replace LIBFCL_LIBRARIES with full path to the library
-find_library(LIBFCL_LIBRARIES_FULL ${LIBFCL_LIBRARIES} ${LIBFCL_LIBRARY_DIRS})
-set(LIBFCL_LIBRARIES "${LIBFCL_LIBRARIES_FULL}")
+include(FindPkgConfig)
```

Comment:
This change is a bit different than the rest of this PR - the rest is removing suppresions for caktin_lint; this appears to be updating the way we use pkg_config from cmake. Seems innocuous though, so I'm happy to have it in this PR.

Reply:
This is related to this PR, because it successfully suppresses another catkin_lint warning as suggested in https://github.com/fkie/catkin_lint/issues/62#issuecomment-462695235.

Reply:
You should keep the original `find_package(PkgConfig REQUIRED)` though. It's the recommended way of using CMake find modules.

Reply:
Thanks for the hint. I thought `include(FindPkgConfig)` and `find_package(PkgConfig REQUIRED)` are synonymous.